src/components/TeamSection/TeamInformation.ts

```diff
@@ -16,18 +23,22 @@ export var executiveTeam: ExecutiveMember[];
+    
+    { id: 2, name: "Fedor Prokopchuk", role: "VP Strategy", image: member_2, linkedin: "https://www.linkedin.com/in/fedor-prokopchuk-707668a6/", email: "mailto:fedor.prokopchuk@ucalgary.ca" },
     { id: 3, name: "Luke Hollinda", role: "VP External", image: member_3, linkedin: "https://www.linkedin.com/in/luke-hollinda-89270a183/?originalSubdomain=ca", email: "mailto:luke.hollinda1@ucalgary.ca" },
-    { id: 4, name: "Amanda Nguyen", role: "VP Internal", image: member_4, linkedin: "https://www.linkedin.com/in/amanda-nguyen-b9146620b/", email: "mailto:amanda.nguyen1@ucalgary.ca" },
+    { id: 4, name: "Amanda Nguyen", role: "VP Community", image: member_4, linkedin: "https://www.linkedin.com/in/amanda-nguyen-b9146620b/", email: "mailto:amanda.nguyen1@ucalgary.ca" },
     { id: 5, name: "Tyler Chan", role: "VP Communications", image: member_5, linkedin: "https://www.linkedin.com/in/chan-tyler/", email: "mailto:tyler.chan@ucalgary.ca" },
-    { id: 6, name: "Sareen Singh", role: "VP Events", image: member_6, linkedin: "https://www.linkedin.com/in/sareen-singh-309b55173/?originalSubdomain=ca", email: "mailto:sareen.singh@ucalgary.ca" }
+    { id: 6, name: "Sareen Singh", role: "VP Events", image: member_6, linkedin: "https://www.linkedin.com/in/sareen-singh-309b55173/?originalSubdomain=ca", email: "mailto:sareen.singh@ucalgary.ca" },
+    { id: 7, name: "Nurgul Akhshatayeva", role: "VP Internal", image: member_7, linkedin: "https://www.linkedin.com/in/nurgul-akhshatayeva-6821b0205/", email: "mailto:nurgul.akhshatayeva@ucalgary.ca" },
+    { id: 8, name: "Markosch Vacil Godwin Saure", role: "VP Finance", image: member_8, linkedin: "https://www.linkedin.com/in/godwin-saure/", email: "mailto:markosch.saure@ucalgary.ca" },
+    { id: 9, name: "Terry Fu", role: "VP Development", image: member_9, linkedin: "https://www.linkedin.com/in/terryfu33/", email: "mailto:terry.fu@ucalgary.ca" },
+    { id: 10, name: "Vidhi Soni", role: "Jr VP External", image: member_10, linkedin: "https://www.linkedin.com/in/vidhi-soni-5ba193196/", email: "mailto:vidhi.soni1@ucalgary.ca" },
+    
 ];
 
 projectManagers = [
-    { id: 7, name: "Daniel Rashidian", project: "Resume Tracker", image: member_7, linkedin: "https://www.linkedin.com/in/armeen-rashidian-b12080195/", email: "mailto:armeenrn@gmail.com" },
-    { id: 9, name: "Darryl Huët", project: "Supply Me", image: member_9, linkedin: "https://www.linkedin.com/in/darrylhuet/", email: "mailto:darryl.huet1@ucalgary.ca" },
-    { id: 8, name: "Felix Vaughan", project: "Social Media Hub", image: member_8, linkedin: "", email: "mailto:ezamabusiness@gmail.com" },
-    { id: 0, name: "Joel Happ", project: "Tech Start Website", image: member_0, linkedin: "https://www.linkedin.com/in/joel-happ/", email: "mailto:joel.happ1@ucalgary.ca" },
+  { id: 11, name: "Mahdi Varposhti", project: " ", image: member_11, linkedin: "https://www.linkedin.com/in/mahdiva/", email: "mailto:mahdi.varposhti1@ucalgary.ca" },
+  
```

Comment:
It doesn't have to be a part of this PR, but sometime soon:

We should add a blank PM with a question mark image, whose name is "You?" and whose project is something like "Apply now at techstartucalgary.com/apply!"